_commands/tools/make.md

```diff
@@ -2,8 +2,115 @@
 ---
 
 make
-----
+-------
 
-<em>TODO: Placeholder</em>
+Simply put, this command can compile code into an executable without having to type a super long compilation sequence. We can use the make utility to automate many tasks in the development process, including compilation and regression testing.
 
+~~~ bash
+$ make main
+c++     main.cpp   -o main
+$ ./main
+~~~
 
+
+<!--more-->
+
+### Useful Options / Examples
+
+In order to do more, we need to create something called a Makefile. There's quite a few tutorials online for how to write a Makefile, but it's usually easiest to grab a Makefile that already exists and modifying the recipes for your project. Take a look at the Makefile below from EECS 280.
+
+Makefiles are comprised of "targets" which execute some task.
+
+~~~ bash
+# Here is a comment!
+target: dependencies
+	[tab] command
+~~~
+
+If you just type `make` without specifying a target, it will run the first target in the file. If you type a specific target name like `make test` it will run the target with that name. If we haven't run the target before or if any of the dependencies have changed, make will go ahead and run whatever command is specified after the tab character on the next line.
+
+#### Example command
+~~~ bash
+$ make Matrix_tests.exe
+g++ --std=c++11 -Wall -Werror -pedantic -O1 Matrix_public_test.cpp Matrix.cpp Matrix_test_helpers.cpp -o Matrix_public_test.exe
+~~~
+
+##### Break it down
+`make Matrix_tests.exe` refers to the target by the same name. This target stitches together all the cpp files needed to compile `Matrix_public_test.exe`.
+
+#### Example command
+~~~ bash
+$ make clean
+rm -vf *.exe *.out.txt *.out.ppm
+Image_public_test.exe
+Image_tests.exe
+Matrix_public_test.exe
+Matrix_tests.exe
+processing_public_tests.exe
+resize.exe
+~~~
+
+##### Break it down
+`make clean` in this Makefile is set up to remove all executables, .out.txtm and .out.ppm files. This is good to have if you want a nice clean start before you compile again.
+
+
+### Final Remarks
+
+Makefiles can save you a lot of time from having to type out long compilation sequences or running test suites!
+
+
+~~~ bash
+# Makefile
+# Build rules for EECS 280 project 2
+
+# Compiler
+CXX = g++
```

Comment:
I don't think it's a good idea to include a definition of CXX in an example makefile. That variable is already pre-defined by make to be set to the default system c++ complier (i.e. on a mac it's set to clang). Setting it explicitly is only useful if you really need a specific compiler for some reason, otherwise it's a bad idea (because this makefile wouldn't work on a mac out-of-the-box)